modules/signals/schematics-core/utility/visitors.ts

```diff
@@ -198,6 +198,86 @@ export function visitDecorator(
   });
 }
 
+export function visitImportDeclaration(
```

Comment:
You will have to move this to the schematics-core project (https://github.com/ngrx/platform/tree/main/modules/schematics-core).
Afterwards you need to run the `copy:schematics` command.

The reasoning behind this is that we share code between our schematics.
This package isn't public, so we do decided it's easier to just copy the schematics in each package. That's also the reason why the current build is failing.

Reply:
Ahhhh, I already wanted to bring up the topic of a shared lib after the 19 release. I think that is not necessary now. Thanks